briskedu/brisk-learning
# Improve logout behavior

Summary:
1. Make user when logged out we don't show Submit Feedback or Logout
2. Make sure data is cleared when logging out

Test Plan:
1. Login and confirm num revisions pops up soon after
2. Confirm Check AI writing works
3. logout
4. Confirm Submit Feedback and Logout don't show up
5. Log back in
6. Confirm num revisions shows up but not AI Check
